Source/Scene/SkyAtmosphere.js

```diff
@@ -80,6 +80,31 @@ define([
         });
         this._spSkyFromSpace = undefined;
         this._spSkyFromAtmosphere = undefined;
+        
+        // hue, saturation, and brightness shift values for color adjustment
+        /**
+         * The hue shift to apply to the atmosphere. Defaults to 0.0 (no shift).
+         * @type {Number}
+         * @default 0.0
+         */
+        this.hueShift = 0.0;
+
+        /**
+         * The saturation shift to apply to the atmosphere. Defaults to 0.0 (no shift).
+         * @type {Number}
+         * @default 0.0
+         */
+        this.saturationShift = 0.0;
+
+        /**
+         * The brightness shift to apply to the atmosphere. Defaults to 0.0 (no shift).
+         * @type {Number}
+         * @default 0.0
+         */
+        this.brightnessShift = 0.0;
+
+        var hsbScratch = new Cartesian3(this.hueShift, this.saturationShift, this.brightnessShift);
+        this._hsbScratch = hsbScratch;
```

Comment:
These two lines can be combined. `_hsbScratch` can be renamed to `hueSaturationBrightness`, just to follow the same format as `cameraAndRadiiAndDynamicAtmosphereColor` below.